DigitalLearningSolutions.Data.Tests/DataServices/CustomPromptsDataServiceTests.cs

```diff
@@ -106,5 +105,26 @@ public void GetPromptNameForCentreAndPromptNumber_returns_expected_prompt_name()
             // Then
             result.Should().BeEquivalentTo("Role type");
         }
+
+        [Test]
+        public void GetCourseCustomPrompts_returns_populated_CourseCustomPromptsResult()
+        {
+            // Given
+            var expectedCourseCustomPromptsResult =
+                CustomPromptsTestHelper.GetDefaultCourseCustomPromptsResult(
+                    1379,
+                    customField1Prompt: null,
+                    customField1Options: "Yes\nNo\nNot sure",
+                    customField1Mandatory: true,
+                    customField2Prompt: null,
+                    customField2Options:"Yes\nNo\nNot sure"

```

Comment:
\r\n? Or is the data different for that setup?

Reply:
This specific record uses \n rather than \r\n. Seems there's a lot of inconsistency between how existing records separate new line data (I think because this is obtained from a textarea element, and if people enter data on a Mac then it will use different ones to a Windows pc).

Reply:
Oh no, that could break a lot of things...

Reply:
Would it? I haven't used custom prompts much, but there was (for a certain amount of time before it was refactored) a similar issue with the centre configuration details. The OrganisationsCovered and GeneralInformation fields were originally (and still are) entered in a text area. They have no formatting suggestions whatsoever, sometimes people would use new lines, sometimes they wouldn't, sometimes it was \r\n, sometimes just \n. That didn't seem to cause any issues there.

Reply:
Just double checked, think it used the same logic as is used in the CustomPrompt constructor. Looks like it should be good to me as it should split on either, unless I am missing something:        
 ```
private List<string> SplitOptionsString(string? options)
        {
            var optionsList = new List<string>();
            if (options != null)
            {
                optionsList.AddRange(options.Split(new [] { '\r', '\n' },
                    System.StringSplitOptions.RemoveEmptyEntries));
            }

            return optionsList;
        }
```